charts/kof-mothership/templates/victoria/service-monitors.yaml

```diff
@@ -17,4 +17,63 @@ spec:
       app.kubernetes.io/instance: cluster
       app.kubernetes.io/name: vmalertmanager
       managed-by: vm-operator
+
+---
+
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
```

Comment:
This better to move to kof-collectors chart under some condition. We don't always install collection or storage on mothership.